docs/program/apis/sessions.md

```diff
@@ -0,0 +1,86 @@
+---
+title: "Session Management with Viam's Client SDKs"
+linkTitle: "Session Management API"
+weight: 20
+type: "docs"
+description: "How to use the session management API with Viam's Client SDKs."
+tags: ["client", "sdk", "viam-server", "networking", "apis", "robot api", "session", "sessions", "session management"]
+---
+
+The session management API provides support for robot session management.
+
+A session:
+
+- allows a client to express that it is actively connected or authenticated to `viam-server` on your robot
+- supports stopping moving components when the connection is no longer active.
+
+### Purpose of session management
+
+When controlling a robot or fleet with Viam, you want a way to understand the presence of the clients that are communicating with and authenticated to each robot's `viam-server` through Viam's various [resource APIs](/program/apis/).
+
+The presence of these clients at one point in time is called a *session*.
+A session technically is any presence mechanism at the RDK application layer maintained by any client with `viam-server`.
```

Comment:
Same here--I think it's true that session management has a whole lot to do with mechanisms for monitoring presence, but session =/= presence mechanism.

Reply:
I had tried to adapt this part but the old docs wording is wild sometimes 😆 i think it's a byproduct of having engineers write docs bc what's technically true might not be particularly easy to understand

Reply:
Yeah I was cross referencing that doc with this as I made my first review pass. We are under no obligation to keep any wording that doesn't make sense :D

Reply:
This sentence is ambiguous with respect to whether it's maintained between a client and `viam-server`, or whether it's maintained, by a client that has `viam-server`.

How about
```suggestion
*Session management* is a presence mechanism between a client and `viam-server`.
```

Reply:
LGTM adding